File: bioweb/src/java/entity/Persona.java
package entity;

/**
 *
 * @author Crispancho
 */
public class Persona {
    
    private int idPersona;
    private String nombre;
    private String tipoPersona;
    private String tipoDocumento;
    private String numeroDocumento;
    private String direccion;
    private String telefono;
    private String correo;
    private String nombreContacto;
    private String numeroContacto;
    
    public Persona() {
        
        
    }

    
    
    public Persona(int idPersona, String nombre, String tipoPersona, String tipoDocumento, String numeroDocumento, String direccion, String telefono, String correo, String nombreContacto, String numeroContacto) {
        this.idPersona = idPersona;
        this.nombre = nombre;
        this.tipoPersona = tipoPersona;
        this.tipoDocumento = tipoDocumento;
        this.numeroDocumento = numeroDocumento;
        this.direccion = direccion;
        this.telefono = telefono;
        this.correo = correo;
        this.nombreContacto = nombreContacto;
        this.numeroContacto = numeroContacto;
    }

    
    public int getIdPersona() {
        return idPersona;
    }

    public void setIdPersona(int idPersona) {
        this.idPersona = idPersona;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getTipoPersona() {
        return tipoPersona;
    }

    public void setTipoPersona(String tipoPersona) {
        this.tipoPersona = tipoPersona;
    }

    public String getTipoDocumento() {
        return tipoDocumento;
    }

    public void setTipoDocumento(String tipoDocumento) {
        this.tipoDocumento = tipoDocumento;
    }

    public String getNumeroDocumento() {
        return numeroDocumento;
    }

    public void setNumeroDocumento(String numeroDocumento) {
        this.numeroDocumento = numeroDocumento;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getNombreContacto() {
        return nombreContacto;
    }

    public void setNombreContacto(String nombreContacto) {
        this.nombreContacto = nombreContacto;
    }

    public String getNumeroContacto() {
        return numeroContacto;
    }

    public void setNumeroContacto(String numeroContacto) {
        this.numeroContacto = numeroContacto;
    }

    @Override
    public String toString() {
        return "Persona{" + "idPersona=" + idPersona + ", nombre=" + nombre + ", tipoPersona=" + tipoPersona + ", tipoDocumento=" + tipoDocumento + ", numeroDocumento=" + numeroDocumento + ", direccion=" + direccion + ", telefono=" + telefono + ", correo=" + correo + ", nombreContacto=" + nombreContacto + ", numeroContacto=" + numeroContacto + '}';
    }
    
}

File: bioweb/entity/Persona.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Persona
 *
 * @author crispancho
 */
class Persona {

    private $idPersona;
    private $nombre;
    private $tipoPersona;
    private $tipoDocumento;
    private $numeroDocumento;
    private $direccion;
    private $telefono;
    private $correo;
    private $nombreContacto;
    private $numeroContacto;
    
    function __construct() {
        
    }

    function getIdPersona() {
        return $this->idPersona;
    }

    function getNombre() {
        return $this->nombre;
    }

    function getTipoPersona() {
        return $this->tipoPersona;
    }

    function getTipoDocumento() {
        return $this->tipoDocumento;
    }

    function getNumeroDocumento() {
        return $this->numeroDocumento;
    }

    function getDireccion() {
        return $this->direccion;
    }

    function getTelefono() {
        return $this->telefono;
    }

    function getCorreo() {
        return $this->correo;
    }

    function getNombreContacto() {
        return $this->nombreContacto;
    }

    function getNumeroContacto() {
        return $this->numeroContacto;
    }

    function setIdPersona($idPersona) {
        $this->idPersona = $idPersona;
    }

    function setNombre($nombre) {
        $this->nombre = $nombre;
    }

    function setTipoPersona($tipoPersona) {
        $this->tipoPersona = $tipoPersona;
    }

    function setTipoDocumento($tipoDocumento) {
        $this->tipoDocumento = $tipoDocumento;
    }

    function setNumeroDocumento($numeroDocumento) {
        $this->numeroDocumento = $numeroDocumento;
    }

    function setDireccion($direccion) {
        $this->direccion = $direccion;
    }

    function setTelefono($telefono) {
        $this->telefono = $telefono;
    }

    function setCorreo($correo) {
        $this->correo = $correo;
    }

    function setNombreContacto($nombreContacto) {
        $this->nombreContacto = $nombreContacto;
    }

    function setNumeroContacto($numeroContacto) {
        $this->numeroContacto = $numeroContacto;
    }


}
